gems/manageiq-providers-amazon/lib/tasks/spec.rake

```diff
@@ -0,0 +1,14 @@
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
+  namespace :test do
+    namespace 'manageiq-providers-amazon' do
+      desc "Setup environment for amazon specs"
+      task :setup => [:initialize, :verify_no_db_access_loading_rails_environment, :setup_db]
+    end
+
+    desc "Run all amazon specs"
+    RSpec::Core::RakeTask.new('manageiq-providers-amazon' => [:initialize, "evm:compile_sti_loader"]) do |t|
+      EvmTestHelper.init_rspec_task(t)
+      t.pattern = FileList['gems/manageiq-providers-amazon/spec/**/*_spec.rb']
+    end
+  end
+end # ifdef
```

Comment:
@matthewd @Fryguy I more or less copied that from the main rake tasks. I think this is the easiest way to run the specs in their own suite for now.
I also tried running the tests more isolated inside the gem directory - https://github.com/durandom/manageiq/commit/b4451310ac8a9aab8c6f5307865d8c08f71d007f but I think that would end up in replicating the gem dependencies and requiring MIQ app by hand. 
So, until we dont have a looser coupling of the provider to MIQ, I guess this is the leanest way